### PR TITLE
SubMark

### PR DESCRIPTION
**Pull request by 0xmra**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Changes in docs 
1. Seprated all the installation commands allowing users to directly copy the commands from GitHub with a single click.
2. Added a separate step to install Meson (sudo apt install meson -y) to ensure it's available for configuring and building Rizin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,25 +24,34 @@ one that is at least version `0.55.0`).
...
+sudo apt install meson -y 
+```
+```
+meson setup build
+```
+```
+meson compile -C build
+```
+```
+sudo meson install -C build
```
Comment (wargio) [inline README.md:41]: there is no real reason to have this splitted.
also add `bash` on the same line to have colored syntax.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ editor able to open disk files, and much more!
 To learn more on Rizin you may want to read the
 [official Rizin book](https://book.rizin.re).
 
-# How to install
+# How to install
```
Comment (wargio) [inline README.md:15]: useless space

File: README.md [hunk 3]
```diff
@@ -24,25 +24,34 @@ one that is at least version `0.55.0`).
...
+```
+```
  -- Thank you for using rizin. Have a nice night!
 [0x00000000]>
-
 ```
 
-To uninstall rizin, execute `sudo ninja -C build uninstall`.
+To uninstall rizin, execute 
+`sudo ninja -C build uninstall`.
```
Comment (wargio) [inline README.md:54]: useless newline.

File: README.md [hunk 4]
```diff
@@ -24,25 +24,34 @@ one that is at least version `0.55.0`).
...
+```
+sudo meson install -C build
 ```
 
 Now you can use `rizin`:
 ```
-$ rizin
+sudo rizin
+```
+```
```
Comment (wargio) [inline README.md:48]: same here. also remove `sudo` since it is not needed.

File: README.md (hunk 1 above)
Comment (0xmra) [reply README.md:41]: > there is no real reason to have this splitted.

😑 yeah why not!

**Pull request closed without merging**